bin/config.js

```diff
@@ -1,19 +1,11 @@
-const rc = require('rc')
+const fsproxy = require('fs-proxy')
 
-module.exports = rc('cordlr', {
```

Comment:
I assume you don't want the command line options to be written to the config file?